pkg/leakybucket/bucket.go

```diff
@@ -218,41 +220,41 @@ func LeakRoutine(l *Leaky) {
 		case <-l.KillSwitch:
 			close(l.Signal)
 			l.logger.Debugf("Bucket externally killed, return")
-			l.AllOut <- types.Event{Overflow: types.SignalOccurence{MapKey: l.Mapkey}, Type: types.OVFLW}
+			l.AllOut <- types.Event{Type: types.OVFLW}
```

Comment:
if we don't return the MapKey, how can the bucket be properly deleted from map ?

Reply:
Sure you're right. And by the way, that wasn't done for some buckets...